Resources/Prototypes/Entities/Mobs/NPCs/slimes.yml

```diff
@@ -0,0 +1,232 @@
+﻿- type: entity
+  name: blue slime
+  id: MobAdultSlimes
+  parent: SimpleMobBase
+  description: It looks so much like jelly. I wonder what it tastes like?
+  suffix: Good
+  components:
+  - type: HTN
+    rootTask: SimpleHostileCompound
+  - type: Sprite
+    drawdepth: Mobs
+    netsync: false
+    sprite: Mobs/Aliens/slimes.rsi
+    layers:
+    - map: [ "enum.DamageStateVisualLayers.Base" ]
+      state: blue_adult_slime
+  - type: Fixtures
+    fixtures:
+    - shape:
+        !type:PhysShapeCircle
+        radius: 0.30
+      density: 80
+      mask:
+      - MobMask
+      layer:
+      - MobLayer
+  - type: MovementAlwaysTouching
+  - type: MobThresholds
```

Comment:
I don't think slimes have space movement in 13?

Reply:
removed